en/assembly/vibration_isolation.md

```diff
@@ -0,0 +1,36 @@
+# Vibration Isolation
+
+This topic shows how to determine whether vibration levels are too high, and lists some simple steps to improve vibration performance.
+
+## Overview
+
+Flight Control boards with in-built accelerometers or gyros are sensitive to vibrations.
+High vibration levels can cause sensor clipping and sensor failures, resulting in a range of problems, from reduced flight efficiency/performance (leading to shorter flights and increased vehicle wear-and-tear), through to fly-aways and crashes.
+
+Airframes are designed to damp/reduce the amplitude of specific structural resonances at the autopilot mounting location.
+Users should further isolate the flight controller from airframe (using foam or some other isolation mount) in order to reduce vibration to the level that the autopilot can handle.
```

Comment:
I would not generally suggest that. Some boards are isolated internally, and in some cases hard-mounting works too.

Reply:
I've generalised this to:

> Further isolation may be needed in order to reduce vibration to the level that sensitive components can handle (e.g. some flight controllers must be attached to the airframe using some form of anti-vibration foam/mount - while others are internally isolated).

FYI, what I'm trying to capture here is Paul's comment:

> Damping as an objective is normally something done to the airframe itself to reduce the amplitude of specific structural resonances at the the autopilot mounting location to levels that the isolation mount + autopilot can handle.